test/unit/spec/PasswordExpired_spec.js

```diff
@@ -294,7 +294,7 @@ function (Q, _, $, OktaAuth, LoginUtil, SharedUtil, Util, PasswordExpiredForm, B
         .then(function (test) {
           expect(test.form.hasErrors()).toBe(true);
           expect(test.form.errorMessage()).toBe(
-            'We found some errors. Please review the form and make corrections.'
+            'oldPassword: The credentials provided were incorrect.'
```

Comment:
this doesn't seem right... the error is a field error, so it should probably show right below the oldPass field, not as a generic error. My guess is that BaseForm needs the errorCauses for field errors. Can you please check?

Reply:
I think that was wrong before and correct now after my change, not the opposite. The correct behavior in this case is that it shows the errorSummary in the errorCause array, since that errorCode doesn't exists in login.properties. Look at the test right after this, it pass the right error as well in errorCauses.

Reply:
https://github.com/okta/okta-signin-widget/blob/master/test/unit/helpers/xhr/PASSWORD_EXPIRED_error_oldpass.js

https://github.com/okta/okta-signin-widget/blob/master/test/unit/helpers/xhr/PASSWORD_EXPIRED_error_complexity.js

Reply:
I wonder then if that is actually wrong is the mock xhr we have... my concern here is that ```oldPassword: The credentials provided were incorrect.``` is the way the server returns a field error (field ```oldPassword``` has an error of ```The credentials provided were incorrect.```

Can you check a real response for this case and see if our mock xhr is missing something? It might be missing an actual errorSummary

Reply:
![screen shot 2017-10-31 at 4 04 08 pm](https://user-images.githubusercontent.com/19613940/32252900-3a7f851e-be55-11e7-9984-9da7845ddffa.png)

Reply:
I think this mock xhr is outdated, right? @mauriciocastillosilva-okta

Reply:
I see. Yes, the mock is not really a true response. Can you update it?

Reply:
yepp!